src/index.ts

```diff
@@ -1,7 +1,11 @@
 export {
     PartialConfig as AwsRumConfig,
-    Orchestration as AwsRum
+    Orchestration as AwsRum,
+    PartialCookieAttributes as PartialCookieAttributes,
+    PageIdFormat as PageIdFormat,
+    Telemetry as Telemetry
```

Comment:
```suggestion
    Orchestration as AwsRum,
    PartialCookieAttributes,
    PageIdFormat,
    Telemetry
```

No need for the `as ...` as the names are not changing